### ISSUE TITLE
Placeholder missing after updating to most recent version?

### ISSUE THREAD

**Issue by jq1106** [muukii/NextGrowingTextView#69]
As the title suggests my placeholder doesn't seem to get set anymore?

**Comment by muukii** [comment]
@jq1106 
Really?
How do you set placeholder?
Could you show me the screen of your application?

**Comment by jq1106** [comment]
`theTextView.placeholderAttributedText = NSAttributedString(string: "Message", attributes: [NSAttributedStringKey.foregroundColor:Theme.inputPlaceholderColor, NSAttributedStringKey.font:self.messageTextView?.textView.font!])`

I didn't take a screenshot and already moved back down to the previous version.

**Comment by muukii** [comment]
@jq1106 
It seems like placeholder is working in the demo application.
What's the difference?

**Comment by jq1106** [comment]
Yeah I tried the demo to see if there was an issue but it worked for me too. Tried looking through the changes in the recent commit to see if I could figure out what caused the issue but couldn't figure it out. All I know is the placeholder works on the 1.2.2 version, but doesn't show if I `pod update` and build my project again.
